pilot/cmd/pilot-discovery/main.go

```diff
@@ -23,6 +23,8 @@ import (
 	"github.com/spf13/cobra"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
+	"flag"
```

Comment:
run goimport please, this should be up with the other golang standard libraries.

Reply:
done